.lintstagedrc

```diff
@@ -1,4 +1,4 @@
 {
     "*.{js,json,css,md,ts,tsx}": ["prettier --write", "git add"],
-    "*.{ts,tsx}": ["npm run lint -- --fix", "git add"],
+    "*.{ts,tsx}": ["npm run lint:fix -- --fix", "git add"],
```

Comment:
Aren't there too many `fix`?

Reply:
Couldn't find a right name. Guess its better to call it like that 60b7d0a9a92fa8a9a32b634abcf23f731877a3da since it lints all packages when committing.